cantabular-import/dp-cantabular-csv-exporter.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.3'
+services:
+    dp-cantabular-csv-exporter:
+        build:
+            context: ../../dp-cantabular-csv-exporter
+            dockerfile: Dockerfile.local
+        command:
+            - reflex
+            - -d
+            - none
+            - -c
+            - ./reflex
+        volumes:
+            - ../../dp-cantabular-csv-exporter:/dp-cantabular-csv-exporter
+        depends_on:
+            - kafka
+        ports:
+            - 26300:26300
+        environment:
+            KAFKA_ADDR:         "kafka:9092"
```

Comment:
Is this needed here:
 BIND_ADDR:          ":26300"

(ask Joshua or David)

Reply:
Yes, I just checked the other services that are part of the cantabular import process and they both include the BIND_ADDR variable, so I've added it in